.travis.yml

```diff
@@ -0,0 +1,43 @@
+sudo: false
+language: python
+python:
+  - 3.7-dev
+  - pypy3
+cache: pip
+
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27-django19
+    - python: 2.7
+      env: TOXENV=py27-django110
+    - python: 3.5
+      env: TOXENV=py35-django19
+    - python: 3.5
+      env: TOXENV=py35-django110
+    - python: 3.6
+      env: TOXENV=py36-django19
+    - python: 3.6
+      env: TOXENV=py36-django110
+    - python: 3.6
+      env: TOXENV=py36-django111
+    - python: pypy
+      env: TOXENV=pypy-django111
```

Comment:
I'm pretty sure we can remove django 1.9 and 1.10 and add django 2.0 and 2.1

Reply:
Ooh +1 we should stop supporting whichever version totally changed how middleware works, I think 1.10